src/translations/en.json

```diff
@@ -36,6 +36,10 @@
         "off": "[%key:state::default::off%]",
         "on": "[%key:state::default::on%]"
       },
+      "battery": {
+        "off": "Normal",
```

Comment:
I'd suggest using "OK" here instead of "Normal". If you do, use the key referencing syntax `[%key:state::plant::ok%]`.

Reply:
Good suggestion, consistency is best!